readme.txt

```diff
@@ -26,11 +26,22 @@ Previously called Yoast ACF Analysis.
 
 == Changelog ==
 
+= 2.0.1 =
+
+Released October 18th, 2017
+
+Bugfixes:
+	* Fixes the mismatch in textdomain according to the plugin slug.
+	* Fixes using an incorrect path when loading plugin data.
+	* Fixes a bug where flexible content in combination with ACF 5.
```

Comment:
Fixes a bug with flexible content and repeaters in combination with ACF 5 causing JS errors